services-js/commissions-search/src/FacetList.js

```diff
@@ -14,47 +17,63 @@ const GET_AREAS = gql`
   }
 `;
 
-class FacetList extends Component {
-  renderArea() {
-    function Area(props) {
-      return (
-        <label className="cb">
-          <input
-            id={`checkbox-area-${props.id}`}
-            name={props.id}
-            type="checkbox"
-            value={props.id}
-            className="cb-f"
-            checked={props.checked}
-            onChange={props.handleCheckChange}
-          />
-          <span className="cb-l">{props.name}</span>
-        </label>
-      );
-    }
+function returnArea(props, policy_area) {
```

Comment:
camel case `policyArea`

Also I don’t understand this function name. Maybe `renderArea` or `renderAreaCheckbox`?